src/components/User/UserAvatar.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+
+import { PreviewCamera, PreviewEmote } from '@dcl/schemas'
+import { WearablePreview } from 'decentraland-ui'
+
+import useProfile from '../../hooks/useProfile'
+
+import './UserAvatar.css'
+
+interface Props {
+  address?: string
+}
+
+export default function UserAvatar({ address }: Props) {
+  const [wearablePreviewController, setWearablePreviewController] = useState<any>()
+
+  const { hasDclProfile } = useProfile(address)
+  const handleLoad = useCallback(() => {
+    setWearablePreviewController(WearablePreview.createController('wearable-preview'))
+  }, [])
+
+  const handleError = useCallback((error) => {
+    console.warn(error)
+  }, [])
+
+  const previewEmote = useMemo(() => {
+    const poses = [PreviewEmote.DAB, PreviewEmote.FIST_PUMP, PreviewEmote.DANCE, PreviewEmote.HEAD_EXPLODE]
+    return poses[(Math.random() * poses.length) | 0]
```

Comment:
I think what you wanted to do here is:

`Math.ceil((Math.random() * poses.length)) || 0`